test/JDBC/expected/BABEL-3395-vu-prepare.out

```diff
@@ -0,0 +1,145 @@
+GO
+~~ROW COUNT: 1~~
+
+INSERT INTO upper_lower_dt(a, b, c, d) values(UPPER(N'比尔·拉'), LOWER(N'比尔·拉'), LOWER(N'比尔·拉'), UPPER(N'比尔·拉'))
+GO
+~~ROW COUNT: 1~~
+
+
+CREATE TABLE upper_lower_text(a TEXT)
+GO
+INSERT INTO upper_lower_text values(CAST ('6F9619FF-8B86-D011-B42D-00C04FC964FF' AS text))
+GO
+~~ROW COUNT: 1~~
+
+
+CREATE TABLE upper_lower_ntext(a NTEXT)
+GO
+INSERT INTO upper_lower_ntext values(CAST ('6F9619FF-8B86-D011-B42D-00C04FC964FF' AS ntext))
+GO
+~~ROW COUNT: 1~~
+
+
+CREATE TABLE upper_lower_image(a IMAGE)
+GO
+INSERT INTO upper_lower_image values(CAST ('6F9619FF-8B86-D011-B42D-00C04FC964FF' AS image))
+GO
+~~ROW COUNT: 1~~
+
+
+-- UPPER
+CREATE VIEW dep_view_upper AS
+select UPPER(a) AS upper_a, UPPER(b) AS upper_b, UPPER(c) AS upper_c, UPPER(d) AS upper_d from upper_lower_dt WHERE UPPER(a) = N'ANIKAIT' and UPPER(b) = N'AGRAWAL' and UPPER(c) = N'ANIKAIT' and UPPER(d) = N'AGRAWAL';
+GO
+
+CREATE TABLE tab_arabic_ci_ai(col varchar(20) COLLATE arabic_ci_ai);
+INSERT INTO tab_arabic_ci_ai VALUES ('COLLATION');
+GO
+~~ROW COUNT: 1~~
+
+
+CREATE TABLE tab_arabic_ci_as(col varchar(20) COLLATE arabic_ci_as);
+INSERT INTO tab_arabic_ci_as VALUES ('COLLATION');
+GO
+~~ROW COUNT: 1~~
+
+
+CREATE TABLE tab_arabic_cs_as(col varchar(20) COLLATE arabic_cs_as);
+INSERT INTO tab_arabic_cs_as VALUES ('COLLATION');
+GO
+~~ROW COUNT: 1~~
+
+
+CREATE TABLE tab_chinese_ci_ai(col varchar(20) COLLATE chinese_prc_ci_ai);
+INSERT INTO tab_chinese_ci_ai VALUES ('COLLATION');
+GO
+~~ROW COUNT: 1~~
+
+
+CREATE TABLE tab_chinese_ci_as(col varchar(20) COLLATE chinese_prc_ci_as);
+INSERT INTO tab_chinese_ci_as VALUES ('COLLATION');
+GO
+~~ROW COUNT: 1~~
+
+
+CREATE TABLE tab_chinese_cs_as(col varchar(20) COLLATE chinese_prc_cs_as);
+INSERT INTO tab_chinese_cs_as VALUES ('COLLATION');
+GO
+~~ROW COUNT: 1~~
+
+
+CREATE PROC dep_proc_upper AS
+select UPPER(a), UPPER(b), UPPER(c), UPPER(d) from upper_lower_dt WHERE UPPER(a) = N'ANIKAIT' and UPPER(b) = N'AGRAWAL' and UPPER(c) = N'ANIKAIT' and UPPER(d) = N'AGRAWAL';
+GO
+
+CREATE FUNCTION dbo.dep_func_upper()
+RETURNS VARCHAR
+AS
+BEGIN
+RETURN (select TOP 1 UPPER(a) from upper_lower_dt);
+END
+GO
+
+-- LOWER
+CREATE VIEW dep_view_lower AS
+select LOWER(a) AS lower_a, LOWER(b) AS lower_b, LOWER(c) AS lower_c, LOWER(d) AS lower_d from upper_lower_dt WHERE LOWER(a) = N'anikait' and LOWER(b) = N'agrawal' and LOWER(c) = N'anikait' and LOWER(d) = N'agrawal';
+GO
+
+CREATE PROC dep_proc_lower AS
+select LOWER(a), LOWER(b), LOWER(c), LOWER(d) from upper_lower_dt WHERE LOWER(a) = N'anikait' and LOWER(b) = N'agrawal' and LOWER(c) = N'anikait' and LOWER(d) = N'agrawal';
+GO
+
+CREATE FUNCTION dbo.dep_func_lower()
+RETURNS VARCHAR
+AS
+BEGIN
+RETURN (select TOP 1 LOWER(a) from upper_lower_dt);
+END
+GO
+
+CREATE FUNCTION dbo.tvp_func_upper_lower() 
+RETURNS TABLE 
+AS 
+RETURN 
+(
+    SELECT CAST (UPPER(a) as VARCHAR) AS upper_a, CAST (LOWER(a) as VARCHAR) AS lower_a
```

Comment:
this test will always be tested only for upgrade from 15.7 or 16.3 to latest. And since upper and lower will be fixed in 15.7 or 16.3, let's remove this CAST to `VARCHAR` on upper and lower.